composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "nosto/module-nostocmp",
   "description": "Nosto Category Merchandising extension for Magento 2",
   "type": "magento2-module",
-  "version": "3.3.0",
+  "version": "3.3.1",
```

Comment:
This bug relates to Magento 2.4.3 if I'm correct. If GraphQL is working on older versions, won't this cause issues on previous versions? Either we need to update this to support only >= 2.4.3 or else we need to provide some sort of backwards compatible fix. Of course, if you are confident that the change is backwards compatible then this is okay.

Reply:
It should be compatible. The interceptor modifies the items of `SearchResultsInterface` which is present in 2.3 version.